README.md

```diff
@@ -2,25 +2,50 @@
 
 ## Goal
 
-Provide a simple web page that allows a user to page through a collection of active user records.
-
-### Assumptions
-* Technology choice is whatever platform you're comfortable in, but try to incoorporate Java into the solution.
-* No CSS styling necessary, seriously!
-* Users per page 5
-* Select 3-4 fields from the user's details to display per page
-* Dynamic paging controls (i.e. if no previous page is available, no previous link is presented)
-* Scalable in terms of not holding the entire user collection in memory, let the API do the work for you.
+In order to increase the efficiency of a client using the FitPay API, we need a new composite
+API resource created that will decrease the number of network calls being made.
+
+### Technology Assumptions
+* Technology choice is whatever platform you're comfortable in
+⋅⋅* If you chose a java based solution, a very basic spring-boot application is included.
 * API credentials and/or bearer tokens can be hard coded with values obtained out-of-band
 * You're familiar with [curl](https://curl.haxx.se) for testing Restful APIs
+* Your solution can ideally be launched with a single command, if not instructions MUST be provided so we can test your new API
+⋅⋅* Example `mvn spring-boot:run`
+
```

Comment:
Should we add a not about submitting the test? Like uploading to a git solution is preferable?

Reply:
I actually don't like things in git, we took that approach in the past and it exposes previous solutions to candidates to "borrow" from.   Not necessarily a good thing for a test.

Reply:
Ahh makes sense